test/test-confirm-email.js

```diff
@@ -0,0 +1,167 @@
+const chai = require("chai");
```

Comment:
Small note on file names. Typically, I've seen them just as `confirm-email.js` _or_ if you want the filename to make it clear it's a test `confirm-email.test.js`